shared/parse/Node.tsx

```diff
@@ -218,9 +219,10 @@ export class Node<C extends Context> {
       return (typeof val === 'function') ? val.toString() : val;
     });
 
+    const dataLine = this.elem.attr('data-line');
     return JSON.stringify({
       ctx: ctxJSON,
-      line: parseInt(this.elem.attr('data-line'), 10),
+      line: parseInt(dataLine === undefined ? '' : dataLine, 10),
```

Comment:
A shorter way of doing these undefined checks is:
`dataLine || ''`

That way, when dataLine is undefined, it defaults to ''